src/control/client/pool.go

```diff
@@ -40,3 +40,16 @@ func (c *connList) CreatePool(req *pb.CreatePoolReq) ResultMap {
 
 	return results
 }
+
+// DestroyPool will Destroy a DAOS pool identified by its UUID.
+func (c *connList) DestroyPool(req *pb.DestroyPoolReq) ResultMap {
+	results := make(ResultMap)
+	mc := c.controllers[0] // connect to first AP only for now
```

Comment:
Is the long-term plan to make these sorts of operations fan-out?

Reply:
yes the way in which the access points are used is still TBD, @liw will be able to elaborate